README.md

```diff
@@ -145,133 +145,66 @@ You can send metrics, histograms, or trace data from your application to the Wav
 * Use [**direct ingestion**](https://docs.wavefront.com/direct_ingestion.html) to send the data directly to the Wavefront service. This is the simplest way to get up and running quickly.
 
-The `WavefrontSender` interface has three implementations:
+Let's [create a `WavefrontClient`](#Sending-Data-via-the-WavefrontClient) to send data to Wavefront either via Wavefront proxy or directly over HTTP.
 
-* Option 1: [Create a `WavefrontProxyClient`](#Option-1-Sending-Data-via-the-Wavefront-Proxy) to send data to a Wavefront proxy.
-* Option 2: [Create a `WavefrontDirectIngestionClient`](#Option-2-Sending-Data-via-Direct-ingestion) to send data directly to a Wavefront service.
-* Option 3: [Create a `WavefrontClient`](#Option-3-Sending-Data-Via-A-WavefrontClient) to send data to either proxies or directly to a Wavefront service.
+> **Deprecated implementations**: *`WavefrontDirectIngestionClient` and `WavefrontProxyClient` are deprecated from proxy version 7.0 onwards. We recommend all new applications to use the `WavefrontClient`.*
 
-### Option 1: Sending Data via the Wavefront Proxy
+### Sending Data via the WavefrontClient
 
->**Prerequisite** <br/>
->Before your application can use a `WavefrontProxyClient`, you must [set up and start a Wavefront proxy](https://docs.wavefront.com/proxies_installing.html).
+Use `WavefrontClientFactory` to create a `WavefrontClient` instance, which can send data directly to a Wavefront service or send data using a Wavefront Proxy.
 
-When sending data via the Wavefront proxy, you need to create a `WavefrontProxyClient`. Include the following information.
+The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` to send multiple Wavefront services.
 
-* The name of the host that will run the Wavefront proxy.
-* One or more proxy listening ports to send data to. The ports you specify depend on the kinds of data you want to send (metrics, histograms, and/or trace data). You must specify at least one listener port. 
-* Optional settings for tuning communication with the proxy.
+### Prerequisites  
+* Sending data via Wavefront proxy? 
+  <br/>Before your application can use a `WavefrontClient` you must [set up and start a Wavefront proxy](https://docs.wavefront.com/proxies_installing.html).  
+* Sending data via direct ingestion? 
+  * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
+  * The URL of your Wavefront instance. This is the URL you connect to when you log in to Wavefront, typically something like `https://<domain>.wavefront.com`.
```

Comment:
Add something along the lines of "you can use http protocol with WavefrontClient to send data to the proxy too (on a single port), but that requires proxy 7.0 or newer".

Reply:
actually to this end, the WavefrontClient will only work over Http since we're using the DDI endpoints - 7.0 is a pre-req to even using the WavefrontClient (realistically)